Sources/Public/LottieConfiguration.swift

```diff
@@ -32,6 +36,19 @@ public struct LottieConfiguration: Hashable {
   /// The decoding implementation to use when parsing an animation JSON file
   public var decodingStrategy: DecodingStrategy
 
+  /// Use the display P3 color space for rendering instead of the default RGB
+  ///  - Defaults to `false`
+  public var useDisplayP3: Bool
+}
+
+extension LottieConfiguration {
+    var renderingColorSpace: CGColorSpace {
+        if let colorspace = CGColorSpace(name: CGColorSpace.displayP3), self.useDisplayP3 {
```

Comment:
a small optimization would be checking `useDisplayP3` first so the colorspace is not created if `useDisplayP3` is `false` (the default behavior)

Reply:
Updated!